pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>software.amazon.cloudformation</groupId>
     <artifactId>aws-cloudformation-rpdk-java-plugin</artifactId>
-    <version>2.0.0</version>
+    <version>2.0.2</version>
```

Comment:
why not 2.0.1?  We haven't released 2.0.1 yet,also for the Python plugin. We can release both with same version.

Reply:
I create other PR to release maven package to 2.0.1:  https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/289

Reply:
Ah for some reason I thought we were up to 2.0.1. I'll kill this, can you please coordinate the release?

Reply:
The 2.0.1 is just released. https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/releases/tag/v2.0.1